### DISCUSSION TITLE
Rounded Corner phones concerns.

### DISCUSSION THREAD

**Discussion by timelessnesses** [ppy/osu#22407] [Q&A] [answered]
Recently I installed osu!Lazer hoping of playing with some cool mods.
But I noticed that osu!Lazer does spaced from my water droplet camera a bit but doesn't spaced from a rounded corner so it's kinda hard to reach something at very edge of the screen.
Also seems like there's no adjust for width and height in mobile.

**Comment by reo-the-enby** [comment]
You can use the screen scaling option in the video options. It works for me on mobile.

**Comment by peppy** [comment, marked as answer]
We already read from manufacturer safe edge specifications, so if it's not automatically padding the game for you, you either have the setting turned off, or your phone manufacturer is not specifying them correctly.

Check the setting "Shrink game to avoid cameras and notches" is turned on. If it already is, as mentioned above by @reo-the-enby, you can manually fix it by setting "Screen scaling" to "everything" and then adjusting the area to fit.
